gtsam/hybrid/HybridFactorGraph.h

```diff
@@ -253,54 +174,43 @@ class HybridFactorGraph : protected FactorGraph<Factor>,
    */
   const DCFactorGraph& dcGraph() const { return dcGraph_; }
 
-  /**
-   * Utility for retrieving the internal gaussian factor graph
-   * @return the member variable gaussianGraph_
-   */
-  const GaussianFactorGraph& gaussianGraph() const { return gaussianGraph_; }
-
-  /**
-   * @brief Linearize all the continuous factors in the HybridFactorGraph.
-   *
-   * @param continuousValues: Dictionary of continuous values.
-   * @return HybridFactorGraph
-   */
-  HybridFactorGraph linearize(const Values& continuousValues) const;
-
   /**
    * @return true if all internal graphs of `this` are equal to those of
    * `other`
    */
   bool equals(const HybridFactorGraph& other, double tol = 1e-9) const;
 
-  /// The total number of factors in the nonlinear factor graph.
-  size_t nrNonlinearFactors() const { return nonlinearGraph_.size(); }
-
   /// The total number of factors in the discrete factor graph.
   size_t nrDiscreteFactors() const { return discreteGraph_.size(); }
 
   /// The total number of factors in the DC factor graph.
   size_t nrDcFactors() const { return dcGraph_.size(); }
 
-  /// The total number of factors in the Gaussian factor graph.
-  size_t nrGaussianFactors() const { return gaussianGraph_.size(); }
-
   /**
    * Clears all internal factor graphs
    * TODO(dellaert): Not loving this!
    */
-  void clear();
+  void clear() {
```

Comment:
Propose to remove this imperative method. Create a new instance!

Reply:
Does not seem to be gone

Reply:
Oh I see what you mean. I thought create new instance meant for each member factor graph.